templates/repo/issue/card.tmpl

```diff
@@ -51,16 +51,18 @@
 		{{end}}
 	</div>
 
-	{{if or .Labels .Assignees}}
-	<div class="extra content labels-list gt-p-0 gt-pt-2">
+	{{if .Labels}}
+	<div class="issue-card-content labels-list">
 		{{range .Labels}}
-			<a target="_blank" href="{{$.Page.RepoLink}}/issues?labels={{.ID}}">{{RenderLabel ctx .}}</a>
+			<a {{if $.Page.PageIsProjects}}target="_blank"{{end}} href="{{$.Page.RepoLink}}/issues?labels={{.ID}}">{{RenderLabel ctx .}}</a>
+		{{end}}
```

Comment:
`{{if $.Page.PageIsProjects}}` should it be `{{if .PageIsProjects}}` 🤔

Reply:
ps: in other places, the "Page" variable is usually called "root" or "ctxData", which means the root context data of this request (aka: `ctx.Data` in backend code)